src/marten/db/query/sql/query.cr

```diff
@@ -86,8 +86,8 @@ module Marten
             @using.nil? ? Model.connection : Connection.get(@using.not_nil!)
           end
 
-          def count
-            sql, parameters = build_count_query
+          def count(column_name : String | Symbol | Nil = nil)
```

Comment:
Can we rename this argument to `field`? 🙏  The reason I am asking this is that this is what will be passed to this method in the end: a field name from which we must determine the corresponding column name.